4.5/windows/windowsservercore/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/windowsservercore
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 4.5.0
+ENV NODE_SHA256 16aab15b29e79746d1bae708f6a5dbed8ef3c87426a9408f7261163d0cda0f56
+
+RUN powershell -Command \
```

Comment:
Thanks, added that in another commit. I have seen `SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]` in https://github.com/docker-library/golang/blob/ba5f6a10dcc46edeb66377261f873e3b61af36f5/1.7/windows/windowsservercore/Dockerfile#L3 which looks quite elegant, but I don't want to change the default shell for users of this Docker image.